src/actions/index.js

```diff
@@ -0,0 +1,20 @@
+let nextBookId = 0;
+
+export const createBook = ({ title, category }) => ({
+  type: 'CREATE_BOOK',
+  book: {
+    id: nextBookId++,
+    title,
+    category,
+  },
+});
+
+export const removeBook = ({ id }) => ({
```

Comment:
We don't need to put `id` inside a `{}` here in the parameter. We can simply:

```javascript
export const removeBook = id => ({ // same })
```

Reply:
I think you missed my reply here https://github.com/nikitin2009/bookstore/pull/1#discussion_r304451240

Reply:
Yeah the project says you should pass the `book` as argument for `handleRemove` in `Book`. But yeah you and I both probably know the instruction probably had a little too many vodka the night before :D :D 

Yeah so we gotta fix that `Book` component `onClick` prop to just pass `book.id` to `handleRemove` instead of the whole `book`.